include/libopencm3/stm32/l4/irq.json

```diff
@@ -81,7 +81,8 @@
         "lcd",
         "aes",
         "rng",
-        "fpu"
+        "fpu",
+        "crs"
```

Comment:
if you're going to do updates for teh L496 and similar, why not add all of them? :)

Reply:
i am working with L433 and was just looking at RM0394 at this moment. can't we add the other parts later?